core/src/main/java/org/apache/accumulo/core/util/ThriftMessageUtil.java

```diff
@@ -42,18 +42,14 @@
   private final TMemoryInputTransport inputTransport;
   private final TCompactProtocol inputProtocol;
 
-  public ThriftMessageUtil() {
-    this(64);
-  }
-
-  public ThriftMessageUtil(int initialCapacity) {
+  public ThriftMessageUtil() throws IOException {
     // TODO does this make sense? better to push this down to the serialize method (accept the
     // transport as an argument)?
-    this.initialCapacity = initialCapacity;
+    this.initialCapacity = 64;
     try {
       this.inputTransport = new TMemoryInputTransport();
     } catch (TTransportException e) {
-      throw new RuntimeException(e);
+      throw new IOException(e);
```

Comment:
It looks like you settled on TTransportException in other places - is IOException intentional here?

Reply:
Yeah. I went with whichever type the calling method handled. Here it handled IOException. It is used by `AuthenticationTokenIdentifier` and `AuthenticationKey` in their respective `write()` and `readFields()` methods, both of which  throw IOException.

The other place I threw TTransportException since the calling methods where already catching IOException and throwing TTransportException.

Reply:
Okay - it just stood out with the other changes - no problem.